cmd/brew-test-bot.rb

```diff
@@ -541,6 +543,8 @@ def download
         puts "#{@tap} HEAD #{tap_revision}"
       end
 
+      ohai "Testing commits #{diff_start_sha1} to #{diff_end_sha1}"
```

Comment:
This looks weird if it is empty: https://dev.azure.com/Homebrew/Homebrew/_build/results?buildId=9315&view=logs&jobId=a5e52b91-c83f-5429-4a68-c246fc63a4f7&taskId=00f564b5-c6ae-5329-43c9-96bc79a72ec9&lineStart=45&lineEnd=45&colStart=1&colEnd=32

Reply:
@MikeMcQuaid same answer as https://github.com/Homebrew/homebrew-test-bot/pull/258#discussion_r283670972

Reply:
I believe Mike is suggesting printing something else more helpful when `diff_start_sha1` or `diff_end_sha1` is undefined, like `Testing commits (undefined) to abcdef0` rather than the current `Testing commits to abcdef0`

Reply:
Yep or just ensuring the values are always set. I would also like to see (as mentioned below) this consolidated or replacing the existing printed values.